tests/common/test_cgroupstelemetry.py

```diff
@@ -313,7 +313,8 @@ def test_telemetry_calculations(self, *args):
             self.assertListEqual(generate_metric_list(cpu_percent_values),
                                  collected_metrics[name]["cpu"]["cur_cpu"][0:5])
 
-    def test_cgroup_tracking(self):
+    @patch("azurelinuxagent.common.osutil.default.DefaultOSUtil._get_proc_stat")
```

Comment:
why do we need to mock getprocstat? thanks

Reply:
This particular unit test does not need the actual data from `/proc/stat` and to make it run on Mac, I mocked the call.

Reply:
would you add a comment explaining that? thanks

Reply:
Done.